stix/common/kill_chains.py

```diff
@@ -204,7 +245,102 @@ class KillChainPhasesReference(stix.EntityList):
     _binding_var = "Kill_Chain_Phase"
     _inner_name = "kill_chain_phases"
 
+    def _is_lmco(self, value):
+        return any(value.phase_id == x.phase_id for x in LMCO_KILL_CHAIN_PHASES)
+
+    def _fix_value(self, value):
+        if isinstance(value, KillChainPhase) and self._is_lmco(value):
```

Comment:
Oh, I see. I never bother including `kill_chain_id`on `KillChainPhaseReference`s (see STIXProject/schemas#241). I guess this is the more conservative approach for now, though perhaps not intuitive (if you try to pass one of your own kill chain phases, it won't work). I'm happy leaving it this way.